MalawiMeta.Api/Endpoints/Cities/MapCityByIdEndpoint.cs

```diff
@@ -0,0 +1,58 @@
+﻿using MalawiMeta.Api.Extensions;
+using MalawiMeta.Api.TransferObjects;
+using MalawiMeta.Api.UseCases.Cities;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.OpenApi.Models;
+
+namespace MalawiMeta.Api.Endpoints.Cities;
+
+public static partial class CityEndpoints
+{
+    private static void MapCityById(this IEndpointRouteBuilder app)
+    {
+        app.MapGet(
+            "/{id}",
+            [ProducesResponseType(typeof(CityResponseDto), StatusCodes.Status200OK)]
```

Comment:
Same feedback as before. It's less cognitive load to read these attributes at the end than beginning